### PR TITLE
Openui5 updates

### PR DESCRIPTION
**Pull request by agraebe**
As discussed: updated the exmaple app fro OpenUI5 and renamed dir from SAPUI5 to official name OpenUI5

### CODE REVIEW

**Comment by agraebe** [comment]
This seems to be related to the directory name change.. Did I miss a dir change in the overall project setup?

**Comment by arthurvr** [comment]
Thanks for your PR!

> This seems to be related to the directory name change.. Did I miss a dir change in the overall project setup?

@samccone Looks like a bug in the CI system. Can you take a look?

File: examples/openui5/bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "todomvc-sapui5",
-  "version": "0.0.0",
+  "name": "todomvc-openui5",
+  "version": "latest",
```
Comment (arthurvr) [inline examples/openui5/bower.json]: Why did you change this?

File: examples/openui5/readme.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# OpenUI5 • [TodoMVC](http://todomvc.com)
+
+OpenUI5 is an Open Source JavaScript UI library, maintained by SAP and available under the Apache 2.0 license. OpenUI5 lets you build enterprise-ready web applications, responsive to all devices, running on almost any browser of your choice. It’s based on JavaScript, using jQuery as its foundation and follows web standards. It eases your development with a client-side HTML5 rendering library including a rich set of controls and supports data binding to different models (JSON, XML and OData).
```
Comment (arthurvr) [inline examples/openui5/readme.md]: Please keep the format of the readme, this was a quote (`>`) ;)

File: index.html [hunk 3]
```diff
@@ -91,6 +91,9 @@
 									<a href="examples/emberjs/" data-source="http://emberjs.com" data-content="Ember is a JavaScript framework for creating ambitious web applications that eliminates boilerplate and provides a standard application architecture.">Ember.js</a>
 								</li>
 								<li class="routing">
+									<a href="examples/openui5/" data-source="http://openui5.org" data-content="OpenUI5 lets you build enterprise-ready web applications, responsive to all devices, running on almost any browser of your choice.">OpenUI5</a>
+								</li>
+								<li class="routing">
```
Comment (arthurvr) [inline index.html]: This is unrelated to the rename so maybe do this in a separated PR (or at least in a separated commit).

**Comment by arthurvr** [comment]
Can you double check this? I can't get the app to work anymore. Maybe there's a missing dependency.

**Comment by arthurvr** [comment]
ping @agraebe :)

**Comment by samccone** [comment]
Friendly ping @agraebe :telescope:

**Comment by agraebe** [comment]
:wave: Hey guys, so sorry! I'm on vacation until next week. I'll work on that - I'm very much looking forward to get UI5 out of experimental mode. Looks like I need to rework quite a bit since there is a new OpenUI5 version out with couple of important things

**Comment by samccone** [comment]
No worries enjoy vacation!!! 🍺⛄️✈️

**Comment by samccone** [comment]
Hey guys, I am going to close the PR, just ping us when you push and think it is good to go. Thanks
